eng/native/init-vs-env.cmd

```diff
@@ -45,14 +45,9 @@ set "__VSCOMNTOOLS="
 set "VSCMD_START_DIR="
 
 :VSDetected
-if "%VisualStudioVersion%"=="16.0" (
-    set __VSVersion=vs2019
-    set __PlatformToolset=v142
-    goto :SetVCEnvironment
-)
 if "%VisualStudioVersion%"=="17.0" (
```

Comment:
FYI Mike - this drops support for v142 and VS2019

Reply:
@mikem8361

Reply:
That is fine with me.